includes/filerepo/backend/FileBackend.php

```diff
@@ -1136,13 +1136,16 @@ protected function doCleanInternal( $container, $dir, array $params ) {
 		}
 		$latest = !empty( $params['latest'] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "latest" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)